### PR TITLE
Added concordance command

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: src/main/scala/org/broadinstitute/hail/utils/richUtils/RichMultiArray2Numeric.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.hail.utils.richUtils
+
+import org.broadinstitute.hail.utils.MultiArray2
+
+class RichMultiArray2Numeric[T](val ma: MultiArray2[T]) extends AnyVal {
+
+  def addElementWise(other: MultiArray2[T])(implicit ev: T => scala.math.Numeric[T]#Ops): MultiArray2[T] = {
```
Comment (cseed) [inline src/main/scala/org/broadinstitute/hail/utils/richUtils/RichMultiArray2Numeric.scala]: I wonder how this `ev` stuff looks in bytecode.  If this is performance critical, specializing the type might be a big improvement vs. a (couple?) virtual function calls.

Comment (tpoterba) [inline src/main/scala/org/broadinstitute/hail/utils/richUtils/RichMultiArray2Numeric.scala]: That's a very good point.  I'll specialize to the types I use at the moment

File: src/main/scala/org/broadinstitute/hail/sparkextras/OrderedOuterJoinRDD.scala [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    }
+    assert(rdd.partitions.length == math.max(nLeftPartitions, nRightPartitions))
+    rdd
+  }
+}
+
+class OrderedOuterJoinRDD[PK, K, V1, V2] private(left: OrderedRDD[PK, K, V1], right: OrderedRDD[PK, K, V2])
+  extends RDD[(K, (Option[V1], Option[V2]))](left.sparkContext, Seq(new OneToOneDependency(left),
+    new OrderedDependency(left.orderedPartitioner, right.orderedPartitioner, right)): Seq[Dependency[_]]) {
+  require(left.partitions.length > 0 && right.partitions.length > 0)
```
Comment (cseed) [inline src/main/scala/org/broadinstitute/hail/sparkextras/OrderedOuterJoinRDD.scala]: I don't quite see where these are necessary.  Won't the empty iterator conditions handle this?  Then you can simplify the 0 conditions in OrderedOuterJoinRDD.apply, too.

Comment (tpoterba) [inline src/main/scala/org/broadinstitute/hail/sparkextras/OrderedOuterJoinRDD.scala]: I think OrderedDependency sorta does the wrong thing in a nPartitions = 0 case.  We should fix that in the future and update this code.

**Pull request merged**